inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape.cpp

```diff
@@ -69,6 +69,23 @@ bool propagateUpperBoundFromExistingDSR(std::shared_ptr<ngraph::Function>& funct
     return function_changed;
 }
 
+void checkDynamicFunction(const ngraph::Function& function) {
```

Comment:
IMO `validateDynamicFunction` would be better.

Reply:
Fixed